app/models/manageiq/providers/vmware/infra_manager/vm/remote_console.rb

```diff
@@ -26,6 +26,7 @@ def remote_console_acquire_ticket_queue(protocol, userid)
       :class_name  => self.class.name,
       :instance_id => id,
       :method_name => 'remote_console_acquire_ticket',
+      :queue_name  => queue_name_for_ems_operations,
```

Comment:
Even though this was queued for the ems_operations role, because we weren't setting the queue name this was being picked up by a priority worker rather than the operations worker